desc/objectives/normalization.py

```diff
@@ -12,6 +12,11 @@ def compute_scaling_factors(thing):
 
     scales = {}
 
+    # change this to also work with Poincare XS where bdry is  ZernikePolynomial
```

Comment:
i dont think we need to do anything to the normalizations. If the LCFS and xsection are compatible than the scale factors that are computed here will be the same for both, so its easier to just compute them from the LCFS

Reply:
I changed it